Tín Chỉ Tích Lũy/Program.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Text;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Tín_Chỉ_Tích_Lũy
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            var reader = new Reader();
+            var listStudent = new List<Student>();
```

Comment:
we can shorten this variable name as students
```suggestion
            var students = new List<Student>();
```